<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationScreen</class>
 <widget class="QDialog" name="RegistrationScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 13, 13);</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>161</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>-10</y>
     <width>161</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="powrot">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>650</y>
     <width>718</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color: white;
max-width:710px;
max-height:82px;
min-width:710px;
min-height:82px;

font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>POWRÓT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>555</y>
     <width>718</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color: white;
max-width:710px;
max-height:82px;
min-width:710px;
min-height:82px;

font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ZAREJESTRUJ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dodajZdjecie">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>460</y>
     <width>718</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>718</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color: white;
max-width:710px;
max-height:82px;
min-width:710px;
min-height:82px;

font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DODAJ ZDJĘCIE</string>
   </property>
  </widget>
  <widget class="QLabel" name="imieLabel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>2</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>IMIĘ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="imie">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>2</y>
     <width>508</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: solid;
border-width:4px;
border-radius:30px;

max-width:500px;
max-height:82px;
min-width:500px;
min-height:82px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="hasloLabel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>272</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>HASŁO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="haslo">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>272</y>
     <width>508</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: solid;
border-width:4px;
border-radius:30px;

max-width:500px;
max-height:82px;
min-width:500px;
min-height:82px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nazwiskoLabel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>92</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NAZWISKO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazwisko">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>92</y>
     <width>508</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: solid;
border-width:4px;
border-radius:30px;

max-width:500px;
max-height:82px;
min-width:500px;
min-height:82px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>182</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>182</y>
     <width>508</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: solid;
border-width:4px;
border-radius:30px;

max-width:500px;
max-height:82px;
min-width:500px;
min-height:82px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="powtHasloLabel">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>362</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>POWTÓRZ HASŁO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="powtHaslo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>362</y>
     <width>508</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>508</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: solid;
border-width:4px;
border-radius:30px;

max-width:500px;
max-height:82px;
min-width:500px;
min-height:82px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHaslo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>741</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>buttonHaslo</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>powrot</zorder>
  <zorder>pushButton</zorder>
  <zorder>dodajZdjecie</zorder>
  <zorder>imieLabel</zorder>
  <zorder>imie</zorder>
  <zorder>hasloLabel</zorder>
  <zorder>haslo</zorder>
  <zorder>nazwiskoLabel</zorder>
  <zorder>nazwisko</zorder>
  <zorder>emailLabel</zorder>
  <zorder>email</zorder>
  <zorder>powtHasloLabel</zorder>
  <zorder>powtHaslo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
